src/instrumentation.ts

```diff
@@ -4,11 +4,18 @@ import indent from "./indent";
 import { Client } from "./pg";
 import { ParsedSettings } from "./settings";
 
-export async function runQueryWithErrorInstrumentation(
+interface InstrumentationError extends Error {
+  severity: string;
+  code: string;
+  detail: string;
+  hint: string;
+}
+
+export async function runQueryWithErrorInstrumentation<T = void>(
```

Comment:
```suggestion
export async function runQueryWithErrorInstrumentation<T = any>(
```

Reply:
Ah yeah ok I see. I'll eslint-ignore that then, yeah?